payjoin/src/receive/mod.rs

```diff
@@ -850,16 +866,17 @@ mod test {
 
         let body = original_psbt.as_bytes();
         let headers = MockHeaders::new(body.len() as u64);
-        UncheckedProposal::from_request(
-            body,
-            "?maxadditionalfeecontribution=182?additionalfeeoutputindex=0",
-            headers,
-        )
+        let min_feerate_param = format!("minfeerate={}", min_feerate.unwrap_or(0));
+        let query = format!(
+            "maxadditionalfeecontribution=182&additionalfeeoutputindex=0&{}",
```

Comment:
this previous query used `?` to separate between pairs, which is not correct. is should be `&` (refer to `url_fromencoded` crate where they split by `&`)

Reply:
Nice catch. This should be its own commit